app/controllers/api/locale.php

```diff
@@ -42,7 +40,8 @@
 
         if ($record) {
             $output['countryCode'] = $record['country']['iso_code'];
-            $output['country'] = (isset($countries[$record['country']['iso_code']])) ? $countries[$record['country']['iso_code']] : $locale->getText('locale.country.unknown');
+            $output['country'] = $locale->getText('countries.'.strtolower($record['country']['iso_code']), $locale->getText('locale.country.unknown'));
```

Comment:
Why did we gave up the isset method check? We better be safe than sorry :)

Reply:
This has multiple instances across the diff, I'll avoid commenting on each one :)

Reply:
`$locale->getText()` takes a default param. So I thought we can use that instead :+1: